KenticoListApp/TodoList.Api/Controllers/ListItemsController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Web.Http;
+using TodoList.Api.Models;
+
+namespace TodoList.Api.Controllers
+{
+    public class ListItemsController : ApiController
+    {
+        private static readonly List<ListItem> SampleItems = new List<ListItem>
+        {
+            new ListItem(new Guid("00000000-0000-0000-0000-000000000000"), "text"),

```

Comment:
You should never return an item with empty Guid. You can go to Tools->Create Guid and create random Guid.